README.md

```diff
@@ -3,7 +3,7 @@
 [![PyPI version](https://img.shields.io/pypi/v/pytest-motor.svg)](https://pypi.org/project/pytest-motor/)
 [![PyPI status](https://img.shields.io/pypi/status/pytest-motor.svg)](https://pypi.python.org/pypi/pytest-motor/)
 [![codecov](https://codecov.io/gh/AustinScola/pytest-motor/branch/master/graph/badge.svg)](https://codecov.io/gh/AustinScola/pytest-motor)
-![https://github.com/AustinScola/pytest-motor/actions/workflows/python.yaml](https://github.com/AustinScola/pytest-motor/actions/workflows/python.yaml/badge.svg)
+![https://github.com/AustinScola/pytest-motor/actions/workflows/test.yaml](https://github.com/AustinScola/pytest-motor/actions/workflows/test.yaml/badge.svg)
```

Comment:
This also means we now don't have information about linting, type checking, and formatting included in a badge.

Reply:
Yeah, but do we need it?

Reply:
I find it useful

Reply:
Maybe add another badge for linting?

```suggestion
![https://github.com/AustinScola/pytest-motor/actions/workflows/test.yaml](https://github.com/AustinScola/pytest-motor/actions/workflows/test.yaml/badge.svg)
![https://github.com/AustinScola/pytest-motor/actions/workflows/lint.yaml](https://github.com/AustinScola/pytest-motor/actions/workflows/lint.yaml/badge.svg)
```